docs/modules/deployment/pages/upgrade/git-upgrade.adoc

```diff
@@ -0,0 +1,84 @@
+[[git-upgrade]]
+= Upgrade {page-component-title} with Git
+
+This tutorial was originally written on the OpenNMS Discourse forum (see https://opennms.discourse.group/t/managing-configuration-changes-with-git/2327[Managing Configuration Changes with Git]).
+
+IMPORTANT: Remember to stop {page-component-title} before beginning an upgrade.
+
+== Commit configuration changes
+
+Run the `git status` command to check if changes to the `etc` directory have been committed.
+If files have been modified, follow the instructions in xref:deployment:upgrade/git-diff.adoc#make-changes[Make your changes] to commit them.
```

Comment:
For my own understanding ... is this a backup of the directory content? We revert to a pristine state, so I want to make sure they aren't losing anything.

Reply:
From what I understand, that's correct. It's a final check for any modified files before an upgrade; if any files have been changed since the user's last commit, they should refer to the "Make changes" section to commit them. :) 

If there's a clearer way to word it, please let me know!